broker-daemon/proto/broker.proto

```diff
@@ -87,6 +89,16 @@ message HealthCheckResponse {
     // The status of the engine, either `'OK'` or the error message returned when accessing it.
     string status = 2;
   }
+
+  /**
+   * OrderbookStatus provides the status of an orderbook for a market.
+   */
+  message OrderbookStatus {
+    // The market for the orderbook, e.g. `BTC/LTC`
+    string market = 1;
+    // The status of the market, either `true` or `false`.
+    bool synced = 2;
```

Comment:
This is different from the other statuses, like `EngineStatus` that return a string with `'OK'` or the error message. This is perhaps more granular, but the inconsistency is weird.